### ISSUE TITLE
Transparent blocks become non-transparent

### ISSUE THREAD

**Issue by Test-Account666** [LoremVIpsum/OpenSimpleFrames#6]
I only tested this on Stairs, but I think they should be transparent too.

I really love Carpenter's Mod and downloaded this instantly when I saw this.

When trying to use transparent blocks like colored glass, the texture the block uses is not transparent.

Normal glass seems to work, but I will also create another issue regarding to that

**Comment by LoremVIpsum** [comment]
theoretically that is easy to fix. will have a look

**Comment by LoremVIpsum** [comment]
ok perhaps slightly harder to fixed, its because normal glass is actually clear, but stained glass is just semi translucent. will continue to look

**Comment by LoremVIpsum** [comment]
![image](https://user-images.githubusercontent.com/119070473/220204777-426b2e0a-3e3c-4343-a964-69fb80ef3992.png)
this is the closest I can get so far

**Comment by Test-Account666** [comment]
That already looks way better :3

I hope you can fix this for modded blocks too, though.

I'll also have a look at the code, maybe I can help :)

**Comment by Test-Account666** [comment]
Also looks like you fixed the xray issue

**Comment by LoremVIpsum** [comment]
have a go with1.0.7 now on modrinth and see what you get (for both your issues)

**Comment by Test-Account666** [comment]
Glass block fixed, it's no longer xray-ing.

Lime glass and modded transparent blocks are still not fixed

![2023-02-21_01 41 21](https://user-images.githubusercontent.com/36412486/220219822-70542e03-9597-4cec-b652-852deeb0beb3.png)

**Comment by TheBlackEntity** [comment]
Fixed in latest version, thanks

**Comment by Test-Account666** [comment]
Wrong issue closed

**Comment by LoremVIpsum** [comment]
Im wondering if this is to do with shaders?

**Comment by Raeffi** [comment]
I tried a few blocks with transparent textures from different mods. I use no shaders.
![2023-03-11_13 35 24](https://user-images.githubusercontent.com/25285430/224484907-9d9f4339-c0b1-4b46-b460-79ac62f799f3.png)

**Comment by Test-Account666** [comment]
> I tried a few blocks with transparent textures from different mods. I use no shaders.
> ![2023-03-11_13 35 24](https://user-images.githubusercontent.com/25285430/224484907-9d9f4339-c0b1-4b46-b460-79ac62f799f3.png)
> 

Did you use Sodium?

I definitely did, so I'm wondering if this might be the cause 🤔

**Comment by Raeffi** [comment]
> Did you use Sodium?
> 
> I definitely did, so I'm wondering if this might be the cause 🤔

My modpack has Sodium, Sodium Extra and a few other optimisation mods

**Comment by LoremVIpsum** [comment]
Yup I imagine there is some sodium based shenanigans, when I was running on my test world I don't think I had it, its surprisingly hard to do transparency consistency with changing blocks lol. when i get a  bit more time I will look into it more,
